ipld/hamt/src/node.rs

```diff
@@ -184,14 +183,13 @@ where
         Q: Eq + Hash,
     {
         let hash = H::hash(q);
-        self.get_value(&mut HashBits::new(&hash), bit_width, 0, q, store)
+        self.get_value(&mut HashBits::new(&hash), bit_width, q, store)
     }
 
     fn get_value<Q: ?Sized, S: Blockstore>(
         &self,
         hashed_key: &mut HashBits,
         bit_width: u32,
-        depth: u64,
```

Comment:
This was probably an artifact from a previous version. We avoid infinite recursion by pulling bits off the hashed key, which has a limited size anyways.